search-sentiments-map/public/scripts/map-script.js

```diff
@@ -142,14 +156,16 @@ function mouseInToRegion(e) {
   let countryData = e.feature.getProperty('country_data');
   // Add popup info window with country info.
   if (countryData != null) {
-    // Set the hover country so the setStyle function can change the border.
+    // Set the hover country so the `setStyle` function can change the
+    // border.
     e.feature.setProperty('country', 'hover');
     countryInfo = e.feature.getProperty('name') + ': ';
 
-    // Display "N/A" on hover when countryData is -500, the value signifying
-    // there were no results.
+    // Display "N/A" on hover when `countryData` is the no results
```

Comment:
I would still have the comment say that there were no results. You want the comment to be as "English" as possible. 
In this project no results implies default score, not the other way round, so you want to say no results.